mne/report.py

```diff
@@ -82,7 +82,11 @@ def _scale_mpl_figure(fig, scale):
         sfactor = -((1. / scale) ** 2)
     for text in fig.findobj(mpl.text.Text):
         fs = text.get_fontsize()
-        text.set_fontsize(fs + sfactor)
+        try:
+            text.set_fontsize(fs + sfactor)
```

Comment:
...is there a reason the scale doesn't just apply a scale factor to the fonts, and instead does something with multiples of four points?